### PR TITLE
#169258599 Travel Admin can create accommodation facility 

### PR DESCRIPTION
**Pull request by victkarangwa**
#### What does this PR do?
This PR adds functionality for a travel admin to create an accommodation facility for users
#### Description of Task to be completed?
- The travel admin is able to enter a name for the location/accomodation facility
- The travel admin is able to enter the location (I.E address) of the center
- The travel admin is able to upload an image of the centre 
- The travel admin is able to create a number of rooms with room names and room type
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-frontend.git`
- `cd team-odd-bn-frontend`
- `git checkout ft-create-accommodation-facility-169258599`
- `npm install` packages
- `npm run dev` to start the development server
- It should open your default browser automatically
- Navigate to `localhost:4000/accommodation/create` to access the page directly
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#169258599](https://www.pivotaltracker.com/story/show/169258599)
###### Adding photos
Screen Shot 2020-01-16 at 11 30 54

###### Adding rooms
Screen Shot 2020-01-20 at 15 30 30

###### creating accommodation
Screen Shot 2020-01-20 at 12 51 50

#### Questions:
N/A

### CODE REVIEW

File: src/__mocks__/accommodation/createAccommodation.js [hunk 1]
```diff
@@ -0,0 +1,321 @@
+// create accommodation reducers
```
Comment (hezronkimutai) [inline src/__mocks__/accommodation/createAccommodation.js]: You can remove this comment victor

File: src/__tests__/containers/accommodation/createAccommodation.test.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+    component
+      .find('[data-test="accommodation-city"]')
+      .simulate('change', { target: { value: 'Kigali' } });
+    expect(handleChangeSpy).toHaveBeenCalled();
+
+
+    component.find('[data-test="addRooms-btn"]').simulate('click');
+    expect(addRoomsSpy).not.toHaveBeenCalled();
+    // component.find('[data-test="addPhotos-btn"]').simulate('click');
```
Comment (hezronkimutai) [inline src/__tests__/containers/accommodation/createAccommodation.test.js]: Remove this comment @victor.

File: src/__tests__/containers/accommodation/createAccommodation.test.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+//       },
+//     };
+
+//     const openCloudinaryUploadWidget = require('./openCloudinaryUploadWidget')
+//       .default;
+
+//     const response = await openCloudinaryUploadWidget().catch((err) => err);
+//     expect(response).toContain(error.message);
+//   });
+// });
```
Comment (hezronkimutai) [inline src/__tests__/containers/accommodation/createAccommodation.test.js]: make sure you remove all the comments

**Review by hezronkimutai — CHANGES_REQUESTED** [review]
Hi @victorkarangwa4 , nice job, correct the responsiveness of small screens

Screen Shot 2020-01-16 at 16 38 52

**Comment by victorkarangwa4** [comment]
> Hi @victorkarangwa4 , nice job, correct the responsiveness of small screens
> 
> Screen Shot 2020-01-16 at 16 38 52

**Review by hezronkimutai — APPROVED** [review]

**Review by NiyongaboEric — CHANGES_REQUESTED** [review]
Good Job @victorkarangwa4 . I can confirm that it works locally. 
Ensure feature is running on Heroku and the footer is not jumping.

File: src/assets/css/accommodation.scss [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+        .address {
+          margin: auto;
+        }
+        .description {
+          margin: auto;
+          textarea {
+            width: 380px;
+          }
+        }
+        //   .upload-btn {
```
Comment (shemaeric) [inline src/assets/css/accommodation.scss]: please remove these commented codes

**Review by shemaeric — CHANGES_REQUESTED** [review]
Good work @victorkarangwa4 , you can work on the requested changes, and also work on inputs so that you leave padding (space) between the left border and the text that will be written. 

Also, make sure that after the admin has successfully created an accommodation, he is redirected to the other page.

**Review by NiyongaboEric — APPROVED** [review]

**Review by nziokaivy — APPROVED** [review]
Great work @victorkarangwa4

**Review by shemaeric — APPROVED** [review]

**Pull request merged**